### DISCUSSION TITLE
Store and Login in App

### DISCUSSION THREAD

**Discussion by shiribailem** [Heroic-Games-Launcher/HeroicGamesLauncher#186] [Ideas]
I suggest this because it seems like it'd be fairly low hanging fruit on programming and have verified a few connected things.

Since the app already uses electron to launch outside pages (store page and compatibility pages), and these pages already load webpages reasonably (I've noticed no issues), it seems like it wouldn't be too hard to leverage this to include a store page option, as well as leverage this during the login process to simplify things.

I've already tried navigating a little around epic in the browser by navigating away from the store page for a game, and have verified that I'm able to log in without issue.

I would also suggest adding a setting switch for built-in-browser vs external browser.

**Comment by flavioislima** [comment]
@tmajibon Added the Store on 1.5.2. The login will be implemented soon as well.

**Comment by flavioislima** [comment]
I'll convert this into a discussion to maybe figure it out the best way to implement a new login on the App.

**Comment by adityaruplaha** [comment]
This is done. @tmajibon can you please mark this as answered?
